.github/workflows/.percy.yml

```diff
@@ -0,0 +1,124 @@
+name: Percy
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "docs/**"
+      - "frontend/test/**"
```

Comment:
The visual tests will be in this directory, right? Should it be in the ignore list?

Reply:
Thanks for catching, fixed that